### PR TITLE
Update core docs

### PR DESCRIPTION
**Pull request by loay**
connect to https://github.com/strongloop/loopback-next/issues/260
cc @bajtos @raymondfeng @ritch @superkhau

### CODE REVIEW

File: packages/core/README.md [hunk 1]
```diff
@@ -3,11 +3,53 @@
...
 - Generate real APIs with a single command
 - Define your data and endpoints with OpenAPI
 - No maintenance of generated code
 
-# License
+
+## Installation
+
+```shell
+$ npm install --save-dev @loopback/core
```
Comment (bajtos) [inline packages/core/README.md]: `--save`

File: packages/core/README.md [hunk 2]
```diff
@@ -3,11 +3,53 @@
...
+```
+
+_This package is typically used in tests, save it to `devDependencies` via `--save-dev`._
```
Comment (bajtos) [inline packages/core/README.md]: This sentence would be appropriate in `testlab` README, not here. `@loopback/core` is used in production.

File: packages/core/README.md [hunk 3]
```diff
@@ -3,11 +3,53 @@
...
+  const app = new Application();
+
+  //Register a controller class with this application.
+
+   class MyController {
+     greet(name: string) {
+       return `hello ${name}`;
+     }
+   }
+   app.route('get', '/greet', operationSpec, MyController, 'greet');
```
Comment (bajtos) [inline packages/core/README.md]: Let's use the example from https://github.com/strongloop/loopback-next/wiki/Hello-World:

```ts
import {Application} from '@loopback/core';

const app = new Application();
app.handler((sequence, request, response) => {
  sequence.send(response, 'hello world'));
});

(async function start() {
  await app.start();
  console.log(`The app is running on port ${app.getSync('http.port')}`);
})();
```

**Comment by loay** [comment]
@bajtos PTAL again .. thanks

**Pull request merged**